### DISCUSSION TITLE
Discussion: Compact sizing for Button controls

### DISCUSSION THREAD

**Discussion by jeganraj-m** [microsoft/microsoft-ui-xaml#7478] [General]
# Discussion: Compact sizing for Button controls
WinUI support compact sizing for textbox, Calendar controls, but not support button controls like Button, DropDown Button, RadioButton, ToggleButton.

Any reason for not providing compact sizing for Button controls or any plan in future?
## Image
![image](https://user-images.githubusercontent.com/74640452/105126046-db777280-5b03-11eb-81e6-8a1f10f3ae50.png)
## Related Links
https://docs.microsoft.com/en-us/windows/uwp/design/style/spacing 
https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/dll/DensityStyles/Compact.xaml

**Comment by mdtauk** [comment]
I am guessing the thinking was that the compact controls are typically long enough to make touching them easier - but, the InfoBar's ActionButton is 24 px high - which is inline with the other controls, so maybe it is time to include compact styles.

![image](https://user-images.githubusercontent.com/7389110/105126579-ff708f00-5ad6-11eb-9300-dc079d248b85.png)


WinUI 3.0 will be required to bring compact support for flyout controls however.

**Comment by jeganraj-m** [comment]
@ranjeshj  Looks like the toggle button still doesn't support compact sizing. Any plans for that? My App looks odd with a normal and toggle button.
